src/features/bivariate_color_manager/atoms/bivariateColorManagerResource.ts

```diff
@@ -5,7 +5,7 @@ import { isApiError } from '~core/api_client/apiClientError';
 import { fillBivariateLegend } from '~utils/bivariate/bivariateLegendUtils';
 import { parseGraphQLErrors } from '~utils/graphql/parseGraphQLErrors';
 import { i18n } from '~core/localization';
-import { createBivariateColorsGraphQLQuery } from '../utils/createBivariateColorsGraphQLQuery';
+import { createBivariateQuery } from '~features/bivariate_manager/utils/createBivariateGraphQLQuery';
 import type { BivariateStatisticsResponse } from '~features/bivariate_manager/types';
```

Comment:
we should rather use relative paths for imports inside feature

Reply:
oh, it's even import of different feature
I'll fix

Reply:
I've moved common part of bivariate features to core/bivariate